src/app/shared/error.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ErrorHandler, Injectable } from "@angular/core";
+import * as Raven from 'raven-js';
+
+let ravenEnabled = false;
+if (process.env.LAUNCHER_SENTRY_DSN) {
+  const config = {
+    environment: process.env.ENV,
+    debug: true
+  };
+  Raven.config(process.env.LAUNCHER_SENTRY_DSN, config).install();
+  ravenEnabled = true;
+}
+
+@Injectable()
+class RavenErrorHandler extends ErrorHandler {
+  handleError(err: any): void {
+    if (!err) {
+      err = new Error("Handling error without actual error, weird...");
```

Comment:
can this happen?

Reply:
I think it shouldn't.. but while testing, I am pretty sure it happened in some weird state using dev webpack (resulting in a err.originalError NPE). So I thought adding that would at least make it clear and report it just in case.